src/audio/volume/volume.c

```diff
@@ -643,7 +643,20 @@ static vol_zc_func vol_get_zc_function(struct comp_dev *dev,
 static void volume_set_alignment(struct audio_stream *source,
 				 struct audio_stream *sink)
 {
-#if SOF_USE_HIFI(3, VOLUME) || SOF_USE_HIFI(4, VOLUME) || SOF_USE_HIFI(5, VOLUME)
+#if SOF_USE_HIFI(5, VOLUME)
+
+	/* Both source and sink buffer in HiFi5  processing version,
+	 * xtensa intrinsics ask for 16-byte aligned.
+	 */
+	const uint32_t byte_align = 16;
+
+	/*There is no limit for frame number, so both source and sink set it to be 1*/
+	const uint32_t frame_align_req = 1;
```

Comment:
Should this come from Kconfig ?
i.e. The selection of HiFi3, HIFI4, HIFI5, AVX etc would set a generic ```CONFIG_FRAME_BYTE_ALIGN``` macro that could be used everywhere ?